egs2/voxceleb/spk1/conf/train_RawNet3.yaml

```diff
@@ -0,0 +1,30 @@
+# RawNet3 reproduce recipe configuration.
+encoder: rawnet3
+#encoder_conf:  # uncomment for customized arguments
+
+#model_conf:    # uncomment for customized arguments
+
+frontend: raw
+
+projector: rawnet3
+projector_conf:
+  input_size: 3072
+  output_size: 256
```

Comment:
Better to also configure the `pooling` in this file, because the `input_size` should be consistent.